hack/jenkins/e2e.sh

```diff
@@ -502,7 +502,12 @@ case ${JOB_NAME} in
   kubernetes-e2e-gce-reboot)
```

Comment:
Maybe update the comment for this one, also noting that we should probably change the name of this job from "reboot"?